Readme.md

```diff
@@ -11,15 +11,21 @@ Simple flavour of node/iojs binary management, no subshells, no profile setup, n
 
 ## Installation
 
+With node/iojs already installed:
+
     $ npm install -g n
 
-or
+or, by cloning this repo and running:
 
-    $ make install
+    $ PREFIX=$HOME make install
 
-to `$HOME`. Prefix later calls to `n` with `N_PREFIX=$HOME`
+to install to `$HOME`. Prefix later calls to `n` with `N_PREFIX=$HOME` or export `N_PREFIX` in your shell initialization file.
```

Comment:
Actually, both `PREFIX` and `N_PREFIX` refer to the _same_ directory and default to `/usr/local/bin`: they both refer to the "prefix directory" into which the *active* node version is installed, and in whose `n` *subdirectory*  `n` is expected to be installed.

The fact that they refer to the same dir. is why I suggested eliminating `PREFIX` from the Makefile and using `N_PREFIX` there as well.